distribution/pom.xml

```diff
@@ -83,6 +83,19 @@
             </plugin>
         </plugins>
     </build>
+    <repositories>
```

Comment:
does this module also require bintray?

Reply:
yes. distribution has the iotdb-server dependency. it will throw error without the repo.
https://travis-ci.org/github/apache/incubator-iotdb/builds/673966792
![image](https://user-images.githubusercontent.com/6711230/79063082-16758b80-7cd2-11ea-94a0-a2c25df42d91.png)